src/stream/src/executor/managed_state/join/join_entry_state.rs

```diff
@@ -12,29 +12,29 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-use std::collections::{btree_map, BTreeMap};
-
+use hashbrown::hash_map::{self, DefaultHashBuilder};
+use hashbrown::HashMap;
```

Comment:
Why hashbrown instead of std hash.

Reply:
std hash does not support custom allocator.